core/src/crds_gossip_push.rs

```diff
@@ -249,11 +254,14 @@ impl CrdsGossipPush {
                         trace!("new_push_messages insert {} {:?}", *p, v);
                         push_messages.entry(*p).or_default().push(v.clone());
                         self.num_pushes += 1;
+                        *self.last_pushed_to.entry(*p).or_insert(0) = now;
```

Comment:
this is invoked for every value which is pushed to this peer. probably would be better just to do it once outside the for loops once `push_messages` is done (line 263). something like:
```
for (pubkey, _) in &push_messages {
   self.last_pushed_to.insert(*pubkey, now);
}
```

Reply:
Yea, I changed to that.